### PR TITLE
Add PassengerCard to ResultDetail

### PR DESCRIPTION
**Pull request by jvaclavik**
- Add PassengerCard to ResultDetail
- Implement Card from `universal-components` in PassengerCard
- Add ExtendedTouchable component for bigger touchable areas in small elements
- Add edit icon to PlaceCard

screenshot 2019-02-27 at 16 29 46

### CODE REVIEW

File: apps/core/scenes/resultDetail/ResultDetail.js [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
+    const dateOfBirth = '22/04/1980';
+    const id = 'DF45SV9';
+    const insurance = 'Travel Inusurance Name';
+    const passengerCount = 1;
+    const bags = [
+      { count: 2, type: '40x15x30cm, 3kg' },
+      { count: 1, type: '55x20x40cm, 8kg' },
+    ];
     return (
       <View style={styles.container}>
```
Comment (JoseDuSV) [inline apps/core/scenes/resultDetail/ResultDetail.js]: What about wrapping it to separate component. I'm worried that after we add other parts like contact details and header this file can grow quite big.

Comment (JoseDuSV) [reply apps/core/scenes/resultDetail/ResultDetail.js]: Maybe I should be more clear. I'm thinking about this whole part which can contain more `PassengerCards` and `add` button.

screenshot 2019-02-27 at 16 45 49

Comment (jvaclavik) [reply apps/core/scenes/resultDetail/ResultDetail.js]: Good idea 👍

Comment (jvaclavik) [reply apps/core/scenes/resultDetail/ResultDetail.js]: Will be implemented in next PR.

**Pull request merged**